### PR TITLE
Fix formatting in kernel demo application files

### PR DESCRIPTION
**Pull request by kar-rahul-aws**
Description
-----------
This PR cleans up the kernel demo application files, using uncrustify tool, so that all the application and header files for FreeRTOS+Kernel Demos use spaces.

Test Steps
-----------
NA

Checklist:
----------

- [ ] <s>I have tested my changes. No regression in existing tests.</s>
- [ ] <s>I have modified and/or added unit-tests to cover the code changes in this Pull Request.</s>

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aggarg — DISMISSED** [review]

File: .github/scripts/core_checker.py [hunk 1]
```diff
@@ -305,11 +305,16 @@
     r'FreeRTOS/Demo/CORTEX_M4F_Infineon_XMC4000_Keil/.*',
     r'FreeRTOS/Demo/AVR_ATMega4809_Atmel_Studio/RTOSDemo/.*',
     r'FreeRTOS/Demo/AVR32_UC3/.*',
+    r'FreeRTOS/Demo/AVR_ATMega4809_IAR/.*',
+    r'FreeRTOS/Demo/AVR_ATMega4809_MPLAB.X/.*',
+    r'FreeRTOS/Demo/AVR_Dx_IAR/.*',
+    r'FreeRTOS/Demo/AVR_Dx_MPLAB.X/.*',
```
Comment (Skptak) [inline .github/scripts/core_checker.py:311]: Isn't this going to skip the copyright check on the source code files we place into these directories?

File: FreeRTOS/Demo/AVR32_UC3/main.c
Comment (Skptak) [inline FreeRTOS/Demo/AVR32_UC3/main.c:1]: This one seems a bit off, the swap from the C++ style comments to the C ones doesn't work super great sometimes
Might be worth formatting this one manually?

File: FreeRTOS/Demo/AVR_ATMega4809_MPLAB.X/main.c [hunk 2]
```diff
@@ -28,67 +28,67 @@ TO MICROCHIP FOR THIS SOFTWARE.
...
+ * "FreeRTOSConfig.h" header file. */
 
-// void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName )
-// {
-//     for( ;; );
-// }
+/* void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName ) */
+/* { */
+/*     for( ;; ); */
+/* } */
```
Comment (Skptak) [inline FreeRTOS/Demo/AVR_ATMega4809_MPLAB.X/main.c:84]:
```suggestion
/* void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName )
 * {
 *     for( ; ; )
 *    {
 *    }
 * }
 */
```

File: FreeRTOS/Demo/AVR_ATMega4809_MPLAB.X/main.c [hunk 3]
```diff
@@ -28,67 +28,67 @@ TO MICROCHIP FOR THIS SOFTWARE.
...
-//     for( ;; );
-// }
+ * This is usefull in application development, for debugging.  To use this
+ * hook, uncomment it, and set configUSE_MALLOC_FAILED_HOOK to 1 in
+ * "FreeRTOSConfig.h" header file. */
+
+/* void vApplicationMallocFailedHook( void ) */
+/* { */
+/*     for( ;; ); */
+/* } */
```
Comment (Skptak) [inline FreeRTOS/Demo/AVR_ATMega4809_MPLAB.X/main.c:94]:
```suggestion
/* void vApplicationMallocFailedHook( void ) */
 * {
 *     for( ; ; )
 *    {
 *    }
 * }
  */
```

File: FreeRTOS/Demo/AVR_Dx_IAR/main.c [hunk 4]
```diff
@@ -28,67 +28,67 @@ TO MICROCHIP FOR THIS SOFTWARE.
...
-//     for( ;; );
-// }
+ * This is usefull in application development, for debugging.  To use this
+ * hook, uncomment it, and set configUSE_MALLOC_FAILED_HOOK to 1 in
+ * "FreeRTOSConfig.h" header file. */
+
+/* void vApplicationMallocFailedHook( void ) */
+/* { */
+/*     for( ;; ); */
+/* } */
```
Comment (Skptak) [inline FreeRTOS/Demo/AVR_Dx_IAR/main.c:94]:
```suggestion
/* void vApplicationMallocFailedHook( void )
 * {
 *     for( ; ; )
 *    {
 *    }
 * }
  */
```

File: FreeRTOS/Demo/AVR_Dx_IAR/main.c [hunk 5]
```diff
@@ -28,67 +28,67 @@ TO MICROCHIP FOR THIS SOFTWARE.
...
+ * "FreeRTOSConfig.h" header file. */
 
-// void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName )
-// {
-//     for( ;; );
-// }
+/* void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName ) */
+/* { */
+/*     for( ;; ); */
+/* } */
```
Comment (Skptak) [inline FreeRTOS/Demo/AVR_Dx_IAR/main.c:84]:
```suggestion
/* void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName )
 * {
 *     for( ; ; )
 *    {
 *    }
 * }
 */
```

File: FreeRTOS/Demo/AVR_Dx_MPLAB.X/main.c [hunk 6]
```diff
@@ -28,67 +28,67 @@ TO MICROCHIP FOR THIS SOFTWARE.
...
+ * "FreeRTOSConfig.h" header file. */
 
-// void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName )
-// {
-//     for( ;; );
-// }
+/* void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName ) */
+/* { */
+/*     for( ;; ); */
+/* } */
```
Comment (Skptak) [inline FreeRTOS/Demo/AVR_Dx_MPLAB.X/main.c:84]:
```suggestion
/* void vApplicationStackOverflowHook(TaskHandle_t *pxTask, char *pcTaskName )
 * {
 *     for( ; ; )
 *    {
 *    }
 * }
 */
```

**Review by Skptak — DISMISSED** [review]

File: .github/scripts/core_checker.py (hunk 1 above)
Comment (kar-rahul-aws) [reply .github/scripts/core_checker.py:311]: That is correct , the copyright is not added correctly for the files in these folders and I plan to update them in a separate PR, this was done for the CI check.

File: FreeRTOS/Demo/AVR32_UC3/main.c
Comment (kar-rahul-aws) [reply FreeRTOS/Demo/AVR32_UC3/main.c:1]: updated the formatting in the comments manually.

**Review by Skptak — APPROVED** [review]

**Review by chinglee-iot — APPROVED** [review]

**Pull request merged**